src/lib.rs

```diff
@@ -43,3 +43,4 @@ mod vade_sidetree;
 #[cfg(feature = "sdk")]
 mod in3_request_list;
 pub use self::vade_sidetree::*;
+pub use vade_sidetree_client::*;
```

Comment:
Can you move the new datatypes to the already existing `datatypes` path?

Also can you only export the components we need?

Just added the `::*` re-export to the datatypes and it showed that we're now also exporting functions from client at `vade-sidetree` (see changes in "Functions" section).

![image](https://user-images.githubusercontent.com/1394421/218469560-a78ad75e-fc4c-491c-a203-efc9ab4a54e2.png)

If we don't need them, then they shouldn't be exported. Also when removing the functions, check if the remaining structs and enums are related to `vade-sidetree` plugin input and/or output. If there are structs/enums that are only used by `vade-sidetree` to make internal requests against the client sub-crate and the user won't have any chance to interact with, then we should omit them as well. Skimming through the current structs, it doesn't look like there are too many internal types, but better check than polluting the datatypes (although the `Error` struct looks like a candidate for such an internal datatype).

Reply:
just changed it to export only selected types from sidetree client .Regarding moving the datatypes to already existing datatype path , i have a doubt because sidetree-client is an independent project originally and we have adapted it to our requirement , if i move types to datatype path , it will make sidetree-client dependent on vade-sidetree ?

Reply:
done changed the sidetree client imports path via datatype and only exporting required types now

Reply:
Had to go for a temporary export of all structs, enums and functions. :/

Client crate will be included in plugin crate in a follow-up ticket and types will be cleaned up from there, using the selective export from this MR as a basis. ^^